config/unicorn.rb

```diff
@@ -0,0 +1,22 @@
+worker_processes Integer(ENV["WEB_CONCURRENCY"] || 1)
+timeout 15
+preload_app true
+
+before_fork do |server, worker|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `server`. You can omit all the arguments if you don't care about them.<br>Lint/UnusedBlockArgument: Unused block argument - `worker`. You can omit all the arguments if you don't care about them.